server/src/test/java/com/netcracker/MessageTest.java

```diff
@@ -0,0 +1,25 @@
+package com.netcracker;
+
+import com.netcracker.domain.Message;
+import org.junit.Test;
+
+import static com.netcracker.domain.MessageType.NAME_REQUEST;
+import static com.netcracker.domain.MessageType.TEXT;
+import static org.junit.Assert.assertEquals;
+
+public class MessageTest {
+    @Test
+    public void shouldReturnMessageTypeAndDataWhenTryToGet() {
+        Message message = new Message(TEXT, "Message text");
+
+        assertEquals(message.getMessageType(), TEXT);
+        assertEquals(message.getData(), "Message text");
+    }
+
+    @Test
+    public void givenMessageWithoutTextShouldReturnNullDataWhenTryToGet() {
```

Comment:
&#x1F34E; **Issue**
You should rename this test and change its contents according to what I wrote above.